torch/_logging/_registrations.py

```diff
@@ -11,6 +15,11 @@
 register_log("dynamic", DYNAMIC)
 register_log("torch", "torch")
 register_log("distributed", DISTRIBUTED)
+register_log(
+    "dist_c10d", ["torch.distributed.distributed_c10d", "torch.distributed.rendezvous"]
+)
+register_log("dist_ddp", ["torch.nn.parallel.distributed"])
+register_log("dist_fsdp", ["torch.distributed.fsdp"])
```

Comment:
I hope we won't add too many top level entries in the future that make TORCH_LOGS hard to use, if that happens we can consider a hierarchical parsing like `distributed.ddp` at that time I guess.

Reply:
Agree, that's why I didn't add so many here. We should at most have 5 or 6 submodule labels.. The current three should cover most debugging experiences IMHO.